src/box/xlog.h

```diff
@@ -649,19 +649,6 @@ struct xlog_tx_cursor
 	size_t size;
 };
 
-/**
- * Create xlog tx iterator from memory data.
- * *data will be adjusted to end of tx
- *
- * @retval 0 for Ok
- * @retval -1 for error
- * @retval >0 how many additional bytes should be read to parse tx
- */
-ssize_t
-xlog_tx_cursor_create(struct xlog_tx_cursor *cursor,
```

Comment:
Now we have `xlog_tx_cursor_destroy` but no `xlog_tx_cursor_create`... Maybe, we could work with `xlog_tx_cursor` somehow?